src/components/Content/PanelComponents.tsx

```diff
@@ -1,12 +1,15 @@
 import { ResourceType } from 'common/enums';
 import UserContentPanel from './UserContentPanel';
+import GroupContentPanel from './GroupContentPanel';
+import ApplicationContentPanel from './ApplicationContentPanel';
+import PolicyContentPanel from './PolicyContentPanel';
 
 // convert to ParentResource?
 const PanelComponents = {
```

Comment:
small suggestion because I know there's a lot of upgrading old code going on. 
for objects like this that you're never mutating, you can use Object.freeze() to ensure they never change